make/autoconf/jvm-features.m4

```diff
@@ -306,7 +306,7 @@ AC_DEFUN_ONCE([JVM_FEATURES_CHECK_GRAAL],
     # Graal is only available where JVMCI is available since it requires JVMCI.
     if test "x$OPENJDK_TARGET_CPU" = "xx86_64"; then
       AC_MSG_RESULT([yes])
-    elif test "x$OPENJDK_TARGET_OS-$OPENJDK_TARGET_CPU" = "xlinux-aarch64"; then
+    elif test "x$OPENJDK_TARGET_CPU" = "xaarch64"; then
```

Comment:
You are missing the same change for JVM_FEATURES_CHECK_JVMCI.
Unless it is done intentionally.

Reply:
It's done a couple lines below: https://github.com/openjdk/jdk/pull/685/files#diff-a09b08bcd422d0a8fb32a95ccf85051ac1e69bef2bd420d579f74d8efa286d2fL343

Or do you mean something else?

Reply:
Uhh. Sorry, I thought JVMCI definition will be first, before Graal and did not look below.